CI Build Fail - RED4ext/Scripting.hpp: No such file or directory

```
error: Scripting.cpp
src\reverse\Scripting.cpp(2): fatal error C1083: Cannot open include file: 'RED4ext/Scripting.hpp': No such file or directory

warning: cannot match target(RED4ext).add_files("vendor\RED4ext\src\red4ext.sdk\**.cpp") at .\xmake.lua:14
warning: add -v for getting more warnings ..
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -18,6 +18,11 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+- name: Checkout submodules
+      run: |
+        git submodule sync --recursive
+        git submodule update --init --force --recursive --depth=1
+
     # Install xmake
     - name: Setup xmake
       uses: xmake-io/github-action-setup-xmake@v1

